src/main/java/use_cases/UserManagerInteractor.java

```diff
@@ -0,0 +1,48 @@
+package use_cases;
+
+import entities.Profile;
+import entities.User;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class UserManagerInteractor {
+
+
+    protected User user;
+
+    public UserManagerInteractor(User user){
+        this.user = user;
+    }
+
+    public List<User> viewLiked(){
+        return user.showLiked();
+    }
+
+    public boolean viewAccountStatus(){
+        return user.showVip();
+    }
```

Comment:
Does viewAccountStatus need to show restriction information?

Reply:
Yes, added separated methods to show restriction and vipstatus. Also added a method to return a string of both restriction status and VIP status

Reply:
> Does viewAccountStatus need to show restriction information?

Yes, added separated methods to show restriction and vipstatus. Also added a method to return a string of both restriction status and VIP status